### PR TITLE
📊 manipulate graph data

### LINKED ISSUE #120: ✂️ manipulate data for graphs
Reshape `aggData` to be used by graphs

### PR DESCRIPTION
**Pull request by astroash**
**Changes**
- manipulate graph data
- add test
- add temp graphs to pages

fixes #120

### CODE REVIEW

File: dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      stacked: true,
+    }],
+  },
+};
+
+export const aggregatedToStackedGraph = (aggData: AggregatedData, unit: DurationUnitEnum) => {
+  const [groupBy, ...labels] = aggData.headers;
+  return {
+    labels,
+    datasets: aggData.rows.map((x, i) => {
```
Comment (eliascodes) [inline dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: More descriptive name: `x` -> `row`

File: dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      const label = x[groupBy];
+      const withoutGroupBy = omit([groupBy], x);
```
Comment (eliascodes) [inline dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: More descriptive name: `withoutGroupBy` -> `rowData` or `rowLogData`

File: dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      const dataObject = map((v) =>
```
Comment (eliascodes) [inline dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: More descriptive name: `dataObject` -> `rowNumericData` or `numericData`?

File: dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        typeof v === 'object'
+          ? toUnitDuration(unit, v)
+          : v
+        , withoutGroupBy) as Dictionary<any>;
```
Comment (eliascodes) [inline dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: No need for this to have an `any`. We know the type is `Dictionary<string | number>`. If you want better type inference try out `Objects.mapValues` from `twine-util`?

Comment (astroash) [reply dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: Would be useful to chat about how much we wanna use ramda with ts. I already pulled this out of a pipe to remove type errors.

Comment (eliascodes) [reply dashboard-app/src/dashboard/dataManipulation/aggregatedToGraphData.ts]: 🤷🏼‍♂️ use it when it's useful, don't use it when it isn't. If you want to trial that type library that supposed to give better inference then go for it.

**Pull request merged**